File: source/Backend/src/main/java/com/acare/backend/dto/agent/AgentRegisterRequest.java
package com.acare.backend.dto.agent;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AgentRegisterRequest {

    private String deviceId;

    // ANDROID / WINDOWS
    private String platform;

    private String hostname;

    private String deviceName;

    private String osVersion;

    private String agentVersion;

    private String appVersion;

    private String username;
}

File: source/MobileApp/app/src/main/java/com/acare/clinic/agent/device/AgentRegisterRequest.kt
package com.acare.clinic.agent.device

data class AgentRegisterRequest(
    val deviceId: String,
    val platform: String = "ANDROID",
    val hostname: String? = null,
    val deviceName: String,
    val osVersion: String,
    val agentVersion: String,
    val appVersion: String,
    val username: String?
)